packages/flutter_tools/lib/src/application_package.dart

```diff
@@ -565,7 +565,9 @@ class ApkManifestData {
     }
 
     final _Element application = manifest.firstElement('application');
-    assert(application != null);
```

Comment:
How about the assert up on line 537? Does it need the same fix?

Reply:
I'm not sure since I don't have the original context. It is possible the dump output never contains fewer than 3 lines, so that prevents test regressions.